sonar-db/src/main/java/org/sonar/db/qualityprofile/ActiveRuleDao.java

```diff
@@ -111,7 +110,7 @@ public void delete(DbSession session, ActiveRuleKey key) {
   }
 
   public List<ActiveRuleParamDto> selectParamsByActiveRuleIds(final DbSession dbSession, List<Integer> activeRuleIds) {
-    return DatabaseUtils.executeLargeInputs(activeRuleIds, new ParamIdToDto(mapper(dbSession)));
+    return executeLargeInputs(activeRuleIds, partitionOfIds -> mapper(dbSession).selectParamsByActiveRuleIds(partitionOfIds));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this lambda with a method reference. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1612)